frontend/cypress/integration/nodepage.spec.js

```diff
@@ -2,7 +2,7 @@
 
 context("Dashboard", () => {
   beforeEach(() => {
-    cy.visit("http://localhost:3000");
+    cy.visit("https://explorer.betanet.near.org");
```

Comment:
local one has some weird issue and can not pass.

Reply:
This is not correct. The tests should test the local deployment, not production. What is the issue you have with your local deployment?

Reply:
will change back and check carefully